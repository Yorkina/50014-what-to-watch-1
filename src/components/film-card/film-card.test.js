import React from 'react';
import renderer from 'react-test-renderer';
import FilmCard from './film-card.jsx';


describe(`FilmCard`, () => {
  it(`should render`, () => {
    const tree = renderer
      .create(<FilmCard
        card={{
          href: `fantastic-beasts-the-crimes-of-grindelwald.html`,
          src: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
          title: `Fantastic Beasts: The Crimes of Grindelwald`,
        }}
        onMouseEnter={jest.fn()}
        onClick={jest.fn()}
      />).toJSON();

    expect(tree).toMatchSnapshot();

  });
});
