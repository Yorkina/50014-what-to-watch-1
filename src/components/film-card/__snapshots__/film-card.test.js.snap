// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmCard should render 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      height={175}
      loop={true}
      muted={true}
      preload="false"
      src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      width={280}
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="fantastic-beasts-the-crimes-of-grindelwald.html"
      onClick={[Function]}
    >
      Fantastic Beasts: The Crimes of Grindelwald
    </a>
  </h3>
</article>
`;
